discovery-provider/src/api/v1/tracks.py

```diff
@@ -461,15 +461,15 @@ def get(self, track_id):
         primary_node = creator_nodes[0]
         signature_param = urllib.parse.quote(json.dumps(signature))
         track_cid = track["track_cid"]
-        if CID_STREAM_ENABLED:
-            if not track_cid:
-                logger.info(
-                    f"We should not reach here! If you see this, it's because the track with id {track_id} has no track_cid. Please investigate."
-                )
-                abort_not_found(track_id, ns)
-            path = f"tracks/cidstream/{track_cid}?signature={signature_param}"
-        else:
+        if not track_cid:
+            logger.info(
```

Comment:
maybe logger.warning or error? & let's prefix this log with something (`tracks:stream`?) to make searching in loggly easier